oci/layout/oci_dest.go

```diff
@@ -300,6 +314,63 @@ func (d *ociImageDestination) CommitWithOptions(ctx context.Context, options pri
 	return os.WriteFile(d.ref.indexPath(), indexJSON, 0644)
 }
 
+// PutBlobFromLocalFileOptions is unused but may receive functionality in the future.
+type PutBlobFromLocalFileOptions struct{}
```

Comment:
```suggestion
type PutBlobFromLocalFileOption struct{}
```
please, multiple options can be independent items at the call site.